memoapp.rb

```diff
@@ -40,28 +38,25 @@
 end
 
 post '/memos' do
-  memos = load_memos
-  id = memos.empty? ? '0' : (memos.keys.map(&:to_i).max + 1)
-  memos[id] = { 'memo_title' => params['memo_title'], 'memo_text' => params['memo_text'] }
-  save_memos(memos)
+  memo_title = params['memo_title']
+  memo_text = params['memo_text']
+  Memo.create(memo_title, memo_text)
```

Comment:
一旦変数にいれる意味があまりないかなと思いました。

```suggestion
  Memo.create(params['memo_title'], params['memo_text'])
```